app/presenters/concerns/candidate_api_data.rb

```diff
@@ -0,0 +1,61 @@
+module CandidateAPIData
```

Comment:
This name feels ambiguous as also have a Candidate API. Do we need a top level `VendorAPI` or even`VendorAPIData` namespace for concerns? Or are these meant to be universal fragments of data?

Reply:
This is how I started but because a lot of the data is reused between the vendor and the register API it made sense to move them out of that directory. I think we can consider renaming when - and if - we refactor the CandidateAPI, but as it is they are easier to find and reuse between vendor and register.